README.md

```diff
@@ -59,19 +73,16 @@ Steps:
 
 This code is tested on GitLab version `11.11.0-ee`.
 
-### Changelogs (Release Notes)
+## Contributing
 
-TagBotGitlab creates a changelog for each release based on the issues that have been closed and the merge requests that have been merged. Unlike [TagBot](https://github.com/JuliaRegistries/TagBot), TagBotGitLab currently does not support custom release notes or customizable templates.
+This package uses the [python-gitlab](https://python-gitlab.readthedocs.io/en/stable/index.html) package to interact with GitLab and it's useful to refer to their documentation when making changes.
+You can test using their API locally by generating a [Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token) and setting it in the `local_test.py` script included.
```

Comment:
```suggestion
You can test using their API locally by generating a [Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token) and setting it in the included `local_test.py` script.
```

Reply:
Might be worth mentioning which scopes are required for the personal access token

Reply:
It depends on what API calls you are trying to test